src/main/kotlin/no/nav/syfo/clients/aktor/AktorService.kt

```diff
@@ -6,15 +6,13 @@ import no.nav.syfo.log
 class AktorService(
     private val aktorregisterClient: AktorregisterClient
 ) {
-    suspend fun getFodselsnummerForAktor(aktorId: AktorId, callId: String) =
-        aktorregisterClient.getNorskIdent(aktorId.value, callId).mapLeft {
-            throw IllegalStateException("Did not find Aktor")
-        }
 
     suspend fun fodselsnummerForAktor(aktorId: AktorId, callId: String): String? {
         var fnr: String? = null
-        getFodselsnummerForAktor(aktorId, callId).mapLeft {
-            val message = "Did not find Fodelsnummer for AktorId"
+        aktorregisterClient.getNorskIdent(aktorId.value, callId).mapLeft {
+            if (it == "Den angitte personidenten finnes ikke") return null // We got a response with no matches
```

Comment:
Denne strengen brukes flere steder, burde være en variabel til gjenbruk for å unngå feil.